cpp/velox/operators/functions/RegistrationAllFunctions.cc

```diff
@@ -67,6 +68,10 @@ void registerFunctionOverwrite() {
       kRowConstructorWithAllNull,
       std::make_unique<RowConstructorWithNullCallToSpecialForm>(kRowConstructorWithAllNull));
   velox::functions::sparksql::registerBitwiseFunctions("spark_");
+  velox::functions::registerBinaryIntegral<velox::functions::CheckedPlusFunction>({"check_plus"});
+  velox::functions::registerBinaryIntegral<velox::functions::CheckedMinusFunction>({"check_minus"});
+  velox::functions::registerBinaryIntegral<velox::functions::CheckedMultiplyFunction>({"check_multiply"});
+  velox::functions::registerBinaryIntegral<velox::functions::CheckedDivideFunction>({"check_divide"});
```

Comment:
Can we just register these functions with spark-aligned names: try_add, try_subtract, try_divide, try_multiply?

Reply:
I think it's better to use names: `check_xx`. As we translate `try_xx` to `try(check_xx)`. if we use name try_xx, then they may be confusing.